pkg/config/config.go

```diff
@@ -101,6 +106,12 @@ type Controller struct {
 	ReportPullingTimeout time.Duration
 	Impersonate          string
 	Gather               []string
+	// DisabledGlobalAnonymizations specifies which of global anonymizations to disable.
+	// By default, we anonymize everything.
+	// To see the detailed info about what we anonymize, go to the docs of package anonymization.
+	DisabledGlobalAnonymizations struct {
+		DisableClusterBaseDomainAnonymization bool
```

Comment:
I guess this should be the opposite. We don't anonymize/obfusscate the IPs and hostnames by default. So I would call it e.g `EnableGlobalObfuscation` or something like that. It would be good to somehow emphasize that this is "only" for IPs and hostnames.